code/modules/vending/autodrobe.dm

```diff
@@ -1,10 +1,10 @@
 /obj/machinery/vending/autodrobe
-	name = "\improper AutoDrobe"
-	desc = "A vending machine for costumes."
+	name = "\improper Freddy Fear's House of Scares"
```

Comment:
This is even more out of character tbh, kek.

Reply:
https://fallout.fandom.com/wiki/Freddy_Fear%27s_House_of_Scares_(company)

Reply:
I mistook AutoDrobe for the normal clothes vendomat, egh.